luarules/gadgets/unit_custom_weapons_cluster.lua

```diff
@@ -307,19 +368,9 @@ function gadget:Initialize()
     end
 end
 
-function gadget:Explosion(weaponDefID, ex, ey, ez, attackerID, projID)
+function gadget:Explosion(weaponDefID, ex, ey, ez, attackerID, _)
```

Comment:
No point keeping unused args at the end, but a name is good in case people copypaste it. Either
```suggestion
function gadget:Explosion(weaponDefID, ex, ey, ez, attackerID, projID)
```
or
```suggestion
function gadget:Explosion(weaponDefID, ex, ey, ez, attackerID)
```